### ISSUE TITLE
Fix Request: AvoidForEachInStreams - do not allow log with non-log statements

### ISSUE THREAD

**Issue by stokpop** [jborgers/PMD-jPinpoint-rules#341]
Log lines are considered good in for each. 

But if there are statements next to logging, it should be "bad".

Example, add the process(l) to this test case in `violation: forEach in streams`:

```java
        letters.stream().forEach(l -> {
            process(l);
            log.info(l);
        });
```
